src/aria2p/cli.py

```diff
@@ -466,11 +484,20 @@ def subcommand_add(api: API, uris: List[str] = None, from_file: str = None) -> i
         from_file: Path to the file to read uris from.
             Deprecated: Every URI that is a valid file-path
             and is not a torrent or a metalink is now read as an input file.
+        options: String of aria2c options to add to download.
+        position: Position to add new download in the queue.
 
     Returns:
         int: 0 if OK else 1.
     """
     uris = uris or []
+    position = position or "0"
+
+    if options:
+        options = {
+            str(opt).strip(): str(val).strip()
```

Comment:
`opt` and `val` are already strings.

```suggestion
            opt.strip(): val.strip()
```

I don't think values could have leading/trailing spaces so it should be safe to strip them (as you did) :+1: